sharktank_models/llama3.1/assets/.gitattributes

```diff
@@ -1,2 +1,6 @@
 toy_llama.irpa filter=lfs diff=lfs merge=lfs -text
 toy_llama.mlir filter=lfs diff=lfs merge=lfs -text
+toy_llama_tp2.rank0.irpa filter=lfs diff=lfs merge=lfs -text
+toy_llama_tp2.rank1.irpa filter=lfs diff=lfs merge=lfs -text
+toy_llama_tp2.mlir filter=lfs diff=lfs merge=lfs -text
+toy_llama_tp2.irpa filter=lfs diff=lfs merge=lfs -text
```

Comment:
Want to switch to `*.irpa` and `*.mlir` instead of adding each individual file?

We are also fairly limited (without spending money) in how much we can store with LFS and download

![image](https://github.com/user-attachments/assets/33645066-97cb-4cef-90f3-f334a553ba4c)

Docs: 
* https://docs.github.com/en/billing/managing-billing-for-your-products/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage
* https://docs.github.com/en/billing/managing-billing-for-your-products/managing-billing-for-git-large-file-storage/upgrading-git-large-file-storage


Just something to be careful with >_>

Reply:
We shouldn't see too much expansion. The biggest issue is that if bandwidth is measured we may encounter issues running on presubmits. 20 MBS of models would be my BS estimate, so we would blow through the limit in 50 PRs which is not that many. Thoughts on how we can consider bandwidth workarounds? I can likely get them a bit smaller but not likely that much tinier.

Reply:
The costs aren't _that_ high, it's just annoying to need to think about it.

50GB of bandwidth per month costs $5/month, $60 per year. That's pennies compared to what we spend on other cloud infra. We'd just need to configure billing in iree-org.

cc @amd-chrissosa 

---

Alternatives are:

* Use regular git instead of git LFS
* Use huggingface
* Use Azure
* Use AWS
* Use GCS

(I'd prefer to use something that community members can upload to, so git LFS or huggingface and not a cloud project)

Reply:
Drive by comment, preference against Git. Git is awful for blob storage and penalizes all checkouts even if the blob isn't being used.  However, we should probably have a standard answer here for blob storage across all projects. IMO Azure, AWS, GCS could be used if we can tie contributor access to credentials. We don't have a good story for that atm though.

Reply:
Yeah, hence trying to use Git LFS... but GitHub sets the free tier laughably low (1GB of storage, 1GB of bandwidth / month) and the pricing model is pay-as-you-go using "data packs"... very strange.

Reply:
Bandwidth usage is at half the quota already x_x
![image](https://github.com/user-attachments/assets/cf0e7619-b2cd-43fa-8989-eb2e1e7493f1)

Let's figure out the billing side offline, @amd-chrissosa ?